test/CMakeLists.txt

```diff
@@ -18,23 +18,27 @@ else()
   message( STATUS "Found Catch2 VERSION ${Catch2_VERSION} DIR ${Catch2_DIR}" )
 endif()
 
+find_package( Boost REQUIRED )
```

Comment:
I think you should list the needed components here

Reply:
This should actually be removed. Thanks